### ISSUE TITLE
Run miniterm with 'python -m serial'

### ISSUE THREAD

**Issue by kylekeppler** [pyserial/pyserial#466]
Easier than remembering `python -m serial.tools.miniterm`.

Simply add:

`__main__.py:`

```
from .tools import miniterm

miniterm.main()
```

Another option would be to add a [setuptools entry point](https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html) which would allow you to simply run `miniterm`.

```
setup(
    ...
    entry_points = {
        'console_scripts': ['miniterm=serial.tools.miniterm:main'],
    }
    ...
)
```

**Comment by ksunden** [comment]
See #360 for some discussion on entry points.

It was suggested in there to do `pyserial-miniterm` to avoid name collisions in $PATH, I meant to go back and update that long ago, but never did, if there is interest, willing to go back and make sure it is current.

**Comment by kylekeppler** [comment]
Appreciate the history. Looks like the short names were implemented in a commit linked in #360 but reverted at some point.

Personally, I prefer a shorter name and the collision issue mentioned is easily solvable by venv or conda environments (not sure how anyone can live without!).

**Comment by zsquareplusc** [comment]
implemented
